llvm/lib/SYCLLowerIR/LowerESIMD.cpp

```diff
@@ -820,7 +820,8 @@ translateSpirvIntrinsic(CallInst *CI, StringRef SpirvIntrName,
   auto translateSpirvIntr = [&SpirvIntrName, &ESIMDToErases,
                              CI](StringRef SpvIName, auto TranslateFunc) {
     if (SpirvIntrName.consume_front(SpvIName)) {
-      Value *TranslatedV = TranslateFunc(*CI, SpirvIntrName);
+      Value *TranslatedV =
+        TranslateFunc(*CI, SpirvIntrName.substr(1, 1));
```

Comment:
Could you add a regression test for this change, please?

Reply:
It is added -  sycl/test/esimd/spirv_intrins_trans.cpp

Reply:
I mean LLVM LIT test, not SYCL LIT test.

Reply:
Oh, I guess you mean IR->IR test. OK, will add. Even though functionality is covered by sycl/test/esimd/spirv_intrins_trans.cpp.